ariadne/src/Ariadne/Wallet/Cardano/Kernel/DB/AcidState.hs

```diff
@@ -266,6 +268,11 @@ createHdAddress :: HdAddress -> Update DB (Either HD.CreateHdAddressError ())
 createHdAddress hdAddress = runUpdate' . zoom dbHdWallets $
     HD.createHdAddress hdAddress
 
+createHdAccount :: HdAccount
```

Comment:
As far as I remember, this does not have to exist or be exposed outside. The replacement is a different function also called `createHdAccount` with the following signature:

```
createHdAccount :: HdAccountId
                -> PrefilteredUtxo
                -> Maybe AccountName
                -> Update DB (Either HD.CreateHdAccountError ())
```

Can you explain where you need this?

Reply:
Yes, but it says `hdAccId`, not `hdAcc`, so it refers to the function with the signature I provided.